Atomic/syscontainers.py

```diff
@@ -671,7 +680,10 @@ def get_template_variables(self, image):
         if manifest_template:
             fd = manifest_template.get_fd()
             with os.fdopen(fd) as f:
-                data = json.loads(f.read())
+                try:
+                    data = json.loads(f.read())
+                except ValueError:
+                    raise ValueError("Invalid manifest.json file in image.")
```

Comment:
Same as above, except this is saved in an ostree layer, which doesn't really help the user much.